internal/backends/postgresql/backend.go

```diff
@@ -64,8 +64,23 @@ func (b *backend) Name() string {
 
 // Status implements backends.Backend interface.
 func (b *backend) Status(ctx context.Context, params *backends.StatusParams) (*backends.StatusResult, error) {
-	// TODO https://github.com/FerretDB/FerretDB/issues/3404
-	return new(backends.StatusResult), nil
+	dbs, err := b.r.DatabaseList(ctx)
```

Comment:
Previous implementation queried `pg_tables` table https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/pg/pgdb/stats.go#L69. This implementation uses metadata from registry.

Reply:
https://github.com/FerretDB/FerretDB/blob/7ca090ff2c5305d5f020bfd4fb555b2175208af7/internal/backends/backend.go#L96-L98

When connection information from `ctx` is already checked, only the cached information is returned.

I think we should get a connection to the first database and call `db.Ping()` on it. If there are no databases – well, it is not that important if PostgreSQL works or not in this case 😆